### ISSUE TITLE
`arguments` keyword not consistently highlighted

### ISSUE THREAD

**Issue by qstrahl** [pangloss/vim-javascript#107]
Example code:
```javascript
function MyDate () {
    Date.apply(this, arguments); // arguments is not highlighted here
}
```

It should be highlighted *anywhere* in a function block.

**Comment by crusoexia** [comment]
global keyword + 1
It is very weird for people to use 'arguments' out of function

**Comment by qstrahl** [comment]
I think I'm favour of that, too. It's easy enough and I doubt anyone would be complaining about it.

**Comment by goatslacker** [comment]
+1

**Comment by qstrahl** [comment]
Closed by #268
